S23/thofstrand/lowerCase.py

```diff
@@ -0,0 +1,91 @@
+import pandas as pd
+
+
+
+def lowerCase(word, lang):
+    word = word.strip()
+    lang = lang.split('-')
+    langStart = lang[0]
+
+    if len(langStart) != 2:
+        return "Invalid Language"
+    
+    if "en" == langStart:
```

Comment:
Why just English?  Shouldn't this be the default for many languages that have case?